tests/transpiler/openqasm3/test_coverage_from_openqasm3.py

```
@@ -0,0 +1,120 @@
+
+
+@pytest.mark.parametrize(("target", "baseline"), AVAILABLE_TARGETS)
+def test_openqasm3_coverage(target, baseline):
+    """Measure openqasm3 -> target coverage over a standard gate set."""
+    ACCURACY_BASELINE = baseline
+    ALLOWANCE = 0.01
+    failures = {}
+    for gate_name in QASM_GATES:
+        try:
+            convert_from_openqasm3_to_x(target, gate_name)
+        except Exception as e:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)